src/endpoints/NewsroomLanguages/types.ts

```diff
@@ -12,7 +12,33 @@ export interface ListResponse {
 export interface SettingsUpdateRequest {
     is_default?: true;
     code?: CultureRef['code'];
-    company_information?: Partial<NewsroomCompanyInformation>;
+    company_information?: {
```

Comment:
maybe we should create a separate type for these fields, but it is up to you

Reply:
I think it's fine for now since this is the only place where it is necessary. If the need arises for more places, we'll extract it into a separate type.